src/advanced_inv_pane.cpp

```diff
@@ -119,7 +119,7 @@ std::vector<advanced_inv_listitem> avatar::get_AIM_inventory( const advanced_inv
         if( worn_item.contents.empty() ) {
             continue;
         }
-        for( std::list<item *> it_stack : item_list_to_stack( worn_item.contents.all_items_top() ) ) {
+        for( const auto &it_stack : item_list_to_stack( worn_item.contents.all_items_top() ) ) {
```

Comment:
Const good, ref good, plsno auto.
```suggestion
        for( const std::list<item *> &it_stack : item_list_to_stack( worn_item.contents.all_items_top() ) ) {
```